Library/Homebrew/cmd/audit.rb

```diff
@@ -659,6 +659,25 @@ def audit_line(line, lineno)
     end
   end
 
+  def audit_prefix_has_contents
+    return unless formula.prefix.directory?
+
+    unless Dir.glob("#{formula.prefix}/**/*").any? do |f|
```

Comment:
What is the `unless` doing here? This code is equivalent to:

```rb
def audit_prefix_has_contents
  return unless formula.prefix.directory?
  unless Dir.glob("#{formula.prefix}/**/*").any? { |f| method_that_acts_on_f(f) }
    # there is no code here!
  end
end

def some_method(f)
  if File.directory?(f)
    false
  else
    f = File.basename(f)
    !(Metafiles.copy?(f) || %w[.DS_Store INSTALL_RECEIPT.json].include?(f))
  end
  problem <<-EOS.undent
    The installation seems to be empty. Please ensure the prefix
    is set correctly and expected files are installed.
    The prefix configure/make argument may be case-sensitive.
  EOS
end
```

which seems unlikely to be the intent...